### PR TITLE
Allow strftime to display time in UTC

### PR DESCRIPTION
**Pull request by willthames**
##### SUMMARY
Update strftime to allow display of UTC time

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
strftime filter plugin

### CODE REVIEW

File: lib/ansible/plugins/filter/core.py [hunk 1]
```diff
@@ -98,14 +98,18 @@ def to_datetime(string, format="%Y-%m-%d %H:%M:%S"):
     return datetime.datetime.strptime(string, format)
 
 
-def strftime(string_format, second=None):
+def strftime(string_format, second=None, UTC=False):
```
Comment (sivel) [inline lib/ansible/plugins/filter/core.py]: To remain consistent with the `now()` function, this should be lowercased.

**Comment by s-hertel** [comment]
@willthames Do you have time to update this? Feel free to reopen and ping us for re-review if you're still interested in this.

**Comment by willthames** [comment]
@s-hertel sure, will do - I don't think I have reopen permissions any longer though

**Comment by willthames** [comment]
@s-hertel this is updated now if you can reopen on my behalf

**Comment by mkrizek** [comment]
@willthames it seems like you cannot reopen PR after the branch was force-pushed or recreated. Google says https://gist.github.com/robertpainsi/2c42c15f1ce6dab03a0675348edd4e2c might help?

**Comment by willthames** [comment]
@mkrizek if you can reopen now (I just pushed the old commit back again), let me know - otherwise I'll open a new PR

**Comment by willthames** [comment]
@s-hertel this is now ready for re-review

**Comment by bcoca** [comment]
note: need to review 'state of timezones' in Python, have not dealt with them since using `pytz` 3rd party lib or building it yourself via tzinfo was the requirement.

In "perfectworld"TM we would not use a `utc=` but a `'tz=utc` parameter and try to get full timezone support, but pending on my 'knowhow refresh of current state of python tz' I'm not sure I would block this as is, since to/from utc tends to be the majority of tz usage.

**Comment by bcoca** [comment]
update:
 >= 3.9 includes 'zoneinfo' which is basically a core version of pytz (not exactly) that uses the IANA tzdata and can be used with the datetime functions. 

For older versions we would need to try to import pytz and then use a totally diff branch as they are very diff apis (zoneinfo works as part of datetime, pytz wraps datetime) ..  something that will not be very maintainable, prone to diff errors/bugs depending on branch.

Leaving the comment here for when in a few years there is 'no python < 3.9) and we can deprecate `utc=` in favor of full `tz=` support and someone comes back to complain on why this was not done in the first place.

**Pull request merged**